config/routes.rb

```diff
@@ -3,6 +3,7 @@
   root 'landing_pages#index'
 
   resources :games, :only => [:new, :create, :show, :index, :update]
+  resources :pieces, :only => [:index, :show, :edit, :update]
 
```

Comment:
We shouldn't need an index action  - that would show all pieces, which we do with the game#show action